wiki/Music_theory/Measure/en.md

```diff
@@ -0,0 +1,15 @@
+---
+stub: true
+tags:
+  - bar
+---
+
+# Measure
+
+A **measure** (or **bar**) in music theory refers to a single unit of time featuring a specific number of [beats](/wiki/Music_theory/Beat) played at a particular [tempo](/wiki/Music_theory/Tempo).
+
+In the [beatmap editor timeline](/wiki/Client/Beatmap_editor/Timelines), a measure can be identified as the gap between two large white ticks:
+
+![A measure in the timing tab timeline](img/measure_timing.png "White ticks represent beats, with the larger ones representing the start of a new measure.")
```

Comment:
this will likely render poorly on the wiki. instead, I'd put the image after the title to illustrate the whole article, and make this sentence complete.

Reply:
moved